openandroidinstaller/styles.py

```diff
@@ -27,4 +27,4 @@ class Markdown(ft.Markdown):
     """Markdown element to replace the markdown element from flet but is selectable."""
 
     def __init__(self, *args, **kwargs):
-        super().__init__(selectable=True, *args, **kwargs)
+        super().__init__(selectable=True, auto_follow_links=True, *args, **kwargs)
```

Comment:
I get the error `TypeError: __init__() got an unexpected keyword argument 'auto_follow_links'` when trying to start OAI with `auto_follow_links=True`...

Reply:
Yeah, sorry about that.... It is fixed now